commands/repocmd.go

```diff
@@ -32,8 +33,8 @@ func NewRepoCommand(ctx context.Context, version string) *cobra.Command {
 	repo := &cobra.Command{
 		Use:     "repo",
 		Aliases: []string{"repository"},
-		Short:   "[Alpha] Manage package repositories.",
-		Long:    "[Alpha] The `repo` command group contains subcommands for managing package repositories.",
+		Short:   "[Alpha]" + repodocs.RepoShort,
+		Long:    "[Alpha]" + repodocs.RepoLong,
```

Comment:
this should probably be `"[Alpha] " + repodocs.RepoXYZ` so we get a space between [Alpha] and the doc comment

Reply:
Good catch. Fixed.